Makefile

```diff
@@ -1,18 +1,53 @@
-TARGETS := $(shell ls scripts)
+.PHONY: all build test validate package package-helm clean
 
-.dapper:
-	@echo Downloading dapper
-	@curl -sL https://releases.rancher.com/dapper/latest/dapper-$$(uname -s)-$$(uname -m) > .dapper.tmp
-	@@chmod +x .dapper.tmp
-	@./.dapper.tmp -v
-	@mv .dapper.tmp .dapper
+all: package
 
-$(TARGETS): .dapper
-	./.dapper $@
+build:
+	@echo "--- Building Webhook Binary ---"
+	@bash -c 'source scripts/version && \
+	mkdir -p bin && \
+	docker buildx build \
+		--file package/Dockerfile \
+		--target binary \
+		--build-arg TARGETOS=linux \
+		--build-arg TARGETARCH=$${ARCH} \
```

Comment:
`TARGETOS` and `TARGETARCH` are built-in and they're derived from the value given by `--platform`.

eg: If you build with `--platform=linux/amd64` then the `TARGETOS` will be linux and `TARGETARCH` will be amd64. Some more information here: https://docs.docker.com/build/building/multi-platform/#cross-compiling-a-go-application.

It's also possible to pass multiple target platform, like `--platform=linux/amd64,linux/arm64` and in that case multiple builds will take place, each with their own `TARGETOS` and `TARGETARCH`.